program/rust/src/utils.rs

```diff
@@ -195,6 +199,25 @@ pub fn is_component_update(cmd_args: &UpdPriceArgs) -> Result<bool, OracleError>
     }
 }
 
+// Return PC_STATUS_IGNORED if confidence is bigger than price divided by MAX_CI_DIVISOR else returns status
+pub fn check_confidence_too_big(
```

Comment:
this function name sounds like it's going to error if the confidence is too big. something more like `get_status_for_update` would be better. (I don't love that name either, but the point being the name should indicate that this returns a status)